scripts/cli/lec.sh

```diff
@@ -256,11 +256,51 @@ LIFERAY_WORKSPACE_HOME="$HOME/.liferay/workspace"
 
 RELEASES_JSON_FILE="${LIFERAY_WORKSPACE_HOME}/releases.json"
 
+RELEASES_JSON_URLS=(
+	"https://releases-cdn.liferay.com/releases.json"
+	"https://releases.liferay.com/releases.json"
+)
+
+_verifyReleasesJsonFile() {
+	if jq . "${RELEASES_JSON_FILE}" &>/dev/null; then
+		return
+	fi
+
+	return 1
+}
 _checkReleasesJsonFile() {
+	if ! _verifyReleasesJsonFile; then
+		rm "${RELEASES_JSON_FILE}" &>/dev/null
+	fi
+
+	local backupJson
+
+	if [[ -f "${RELEASES_JSON_FILE}" ]]; then
+		backupJson="$(cat "${RELEASES_JSON_FILE}")"
+	fi
+
+	for url in "${RELEASES_JSON_URLS[@]}"; do
```

Comment:
will this compare url with each entry in `RELEASES_JSON_URLS`? or will this compare with a single string composed of all the entries in `RELEASES_JSON_URLS`?